src/ralph/data_center/models/physical.py

```diff
@@ -669,6 +669,11 @@ def _validate_slot_no(self):
                         'slot_no': message
                     })
 
+    def _validate_hostname(self):
```

Comment:
Why not just strip whitespaces silently? They are certainly an unwanted side effect of copy-paste. Maybe we could use `_validate_hostname` to verify that the hostname complies with appropriate RFC, and just strip leading and trailing whitespaces silently in `clean`?